linux-functions.sh

```diff
@@ -122,23 +106,21 @@ EOF
     # TODO(benkraft): Pull the version we want from webapp somehow.
     # curl for various scripts (including setup.sh)
     sudo apt-get install -y git \
-        python-dev \
+        python-dev python3-dev python3-pip pipenv \
         python-mode python-setuptools \
+        gcc \
```

Comment:
interesting, where'd this come from? Or perhaps more interestingly, why didn't we have it before? I'd wager it was included as a dependency or something?

Reply:
We had it before.  I just did some totally unnecessary reordering...

Reply:
Doh! I thought I did a match on all the packages and this was the only discrepancy not related to removing python 2. Guess I must have missed the original, sorry for any confusion!